### ISSUE TITLE
Give hints when unpublished games exist but are not returned in search results

### ISSUE THREAD

**Issue by charlydelta** [lutris/website#99]
I just tried to look up the game "Battletoads & Double Dragon" (old SNES game).

The search for "battletoads" returned [0 results](https://lutris.net/games/?q=battletoads), so I decided to submit the game to the database. After filling everything in and hitting the submit button, a message appeared that the game already existed, giving me the [corresponding link](https://lutris.net/games/battletoads-double-dragon/).

At first I thought someone was faster than me, but it turns out that the search still results in [0 entries found](https://lutris.net/games/?q=battletoads).

~~I am not yet sure if this is correlated to the `&` symbol, since the search does not find "Battletoads in Battlemaniacs" as well. I will now try to submit this game, too. If submitting the game works, this theory is most likely true.~~

**Edit**: Seems like my theory was wrong and the real reason for the game not displaying was that it still was in the moderation pipeline.

~~I will comment again after I tried submitting "Battletoads in Battlemaniacs".~~

**Comment by christiandreher** [comment]
Submitting worked, so the reason for the malfunctioning could be related to the `&` symbol.

**Comment by charlydelta** [comment]
So, what if we just add a notice when a user searches for a game? Lets say that "Game X" was already submitted, but not yet moderated/published.
If I now search for "Game X", the webclient could display a note saying something like:

```python
"""
Your search resulted in no matches. However, there are {count_unpublsihed}
games that meet your search criteria which are yet to be published.
<a href="?published=false">Show unpublished games</a>.
""".format(count_unpublsihed=len(unpublished_games))
```

**Comment by charlydelta** [comment]
Implemented the template-part of this in #105.

**Comment by charlydelta** [comment]
Implemented and improved this in #115
